webapp/package.json

```diff
@@ -89,7 +88,8 @@
     "ual-lynx": "^0.4.0",
```

Comment:
@tetogomez  I think we can keep anchor as the only UAL provider , these wallets are rarely used

    "ual-lynx": "^0.4.0", - last updated 4 years ago
    "ual-meetone": "^0.1.1", - last updated 4 years ago
    "ual-scatter": "^0.3.0", - not needed and not being updated
    "ual-token-pocket": "^0.3.0", - last updated 2 years ago

Reply:
got it!